streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -287,11 +287,22 @@
      * @param windows the specification of the aggregation window
      * @param <T>   the value type of the aggregated table
      */
-    <T, W extends Window> KTable<Windowed<K>, T> aggregateByKey(Aggregator<K, V, T> aggregator,
+    <T, W extends Window> KTable<Windowed<K>, T> aggregateByKey(T initValue,
```

Comment:
That is a good point, another option would be also using a initialValueSupplier lambda with

```
T apply(K key)
```

or even

```
T apply()   // without key, assuming all initial value will be the same
```

@ymatsuda Thoughts?